guide/additional-info/changes-in-v14.md

```diff
@@ -531,6 +532,16 @@ A new `ignore` event has been added which is emitted whenever an element is not
 
 `CommandInteraction#commandGuildId` has been added which is the id of the guild the invoked application command is registered to.
 
+### ForumChannel
+
+Added class `ForumChannel` which represents GuildForum.
```

Comment:
```suggestion
discord.js supports forum channels as of 14.4.0.
```

Reply:
Maybe for me 
But it looks little strange in guide 
![image](https://user-images.githubusercontent.com/76836550/191279957-6e70d213-577f-4d68-bb1d-a09b47b08a66.png)

Reply:
`Added support for forum channels as of 14.4.0.` sounds better?
`

Reply:
Sure.